### PR TITLE
Add the code generation of injection

### PR DESCRIPTION
**Pull request by houshengbo**
* This PR also adds the generation of "injection" for further reconciler.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -12,6 +12,8 @@ required = [
   "github.com/knative/test-infra/scripts",
   "github.com/knative/test-infra/tools/dep-collector",
   "sigs.k8s.io/controller-tools/pkg/crd/generator",
+  # For load testing.
+  "github.com/rakyll/hey"
```
Comment (greghaynes) [inline Gopkg.toml]: Do we need to add this dep, is it being used anywhere here?

Comment (houshengbo) [reply Gopkg.toml]: PLZ check the issue #57.

**Comment by k4leung4** [comment]
https://github.com/knative/test-infra/issues/770

This can be manually overrided with "cla: yes"
https://github.com/knative/test-infra/issues/770#issuecomment-494506556

**Comment by houshengbo** [comment]
cla: yes

**Comment by k4leung4** [comment]
@adrcunha Do you know why the CLAbot is complaining about "author consent"?

**Comment by adrcunha** [comment]
> @adrcunha Do you know why the CLAbot is complaining about "author consent"?

No. Commits were force pushed 6 times, then the error happened. Then there were 2 more force pushes, and the latest one seems OK. My theory is that one of the 6 force pushes had a wrong user. However, since there's no history to prove or reject my theory, it's just a theory. I'll force the CLA and let's move on.

**Comment by k4leung4** [comment]
It seems like @adrcunha theory might be correct, that some of the force commits may have overridden ones with co-authors, which we might not be able to bypass.

@adrcunha to confirm.

**Comment by adrcunha** [comment]
> It seems like @adrcunha theory might be correct, that some of the force commits may have overridden ones with co-authors, which we might not be able to bypass.
> @adrcunha to confirm.

As I said, I can't confirm it because there's no history. But the new label should solve the issue anyways.

**Comment by k4leung4** [comment]
@googlebot rescan

**Comment by adrcunha** [comment]
@googlebot I consent.

**Pull request merged**